example/pyramid.cpp

```diff
@@ -34,8 +34,7 @@ int main() {
     int min_height = *std::min_element(heights.begin(), heights.end());
 
     // Build rings, diminishing up to pyramid height
-    mcpp::Coordinate base_pt = heights.base_pt();
-    base_pt.y = min_height;
+    mcpp::Coordinate base_pt = {heights.base_pt(), min_height};
```

Comment:
Not sure how I feel about this constructor... ordering seems important for an ordered x,y,z coordinate